CourseGrab/SceneDelegate.swift

```diff
@@ -48,10 +48,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                     // If both are LoginViewController, don't bother presenting a new VC
                     return
                 }
-
+                
                 newVC.modalPresentationStyle = .overFullScreen
                 newVC.modalPresentationCapturesStatusBarAppearance = true
-                currentVC.present(newVC, animated: true)
+                // Only present LoginViewController modally
```

Comment:
Is this accurate? It will also display HomeVC modally over LoginVC, no?

Reply:
Yup, in that case we want it to do that. Think of it as a session, when you sign in or sign out, the user will enter or exit the session modally but once you're in even if you close the app on your phone it shouldn't present modally.